frontend/src/components/Common/LocationHelper.js

```diff
@@ -31,6 +33,7 @@ export async function getNearbyRestaurants(coordinates,
     radius: radius,
     type: ['restaurant'], // Default value
     keyword: keyword,
+    maxPriceLevel: maxPriceLevel,
```

Comment:
what happens in a case where a restaurant doesn't have a pricelevel? Does this get returned with undefined or are they filtered out automatically.

Reply:
Not sure, but I'd guess that the API just wouldn't return that restaurant in the results if a price level is specified

Reply:
I guess there would be no such case since the value is passing from the form but maybe I can implement validation in the function to prevent these accidents?

Reply:
nah if its the case like seif described above, it should be algss